### PR TITLE
Add support for async templates

### PR DESCRIPTION
**Pull request by akx**
Simply: allow a `template` function to return a promise of a string.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -106,6 +106,21 @@ test('custom template', () => {
 	});
 });
 
+test('custom async template', () => {
+	return compiler(
+		{},
+		getConfig({
+			context: { title: 'Pizza', htmlAttributes: { lang: 'it' } },
```
Comment (bebraw) [inline test.js]: Maybe drop `htmlAttributes` since it's not used in this test.

Comment (akx) [reply test.js]: Ah yeah, I just copy-pasted it...

File: README.md [hunk 2]
```diff
@@ -93,7 +93,9 @@ const config = {
 
 Use [@vxna/mini-html-webpack-template](https://www.npmjs.com/package/@vxna/mini-html-webpack-template) to add an app container div, a favicon, meta tags, inline JavaScript or CSS.
 
-Or define a template function to generate your own code:
+Or define a template function to generate your own code.
+
+The template function may return a string or a promise resolving to a string.
```
Comment (sapegin) [inline README.md]:
```suggestion
The template function may return a string or a Promise resolving to a string.
```

**Comment by bebraw** [comment]
@sapegin Any idea why the semantic release bot didn't catch this one?

**Comment by sapegin** [comment]
You can always check the logs ;-)

**Comment by sapegin** [comment]
There's no tag in the commit.

**Comment by bebraw** [comment]
Ah, I see. That's what the 🗣was about at #20 then?

What's the correct way to trigger the bot manually?

**Comment by sapegin** [comment]
For example, make and empty commit (`--allow-empty`) with a correct tag (`feat: XXXX`).

**Comment by bebraw** [comment]
Ok, got it. Just realized it wasn’t following the commit convention.

**Pull request merged**